ReSharper.FSharp/test/src/FSharp.Intentions.Tests/src/QuickFixes/RemoveRedundantAttributeTest.fs

```diff
@@ -1,8 +1,11 @@
 ﻿namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Intentions.QuickFixes
 
+open JetBrains.ReSharper.Plugins.FSharp.ProjectModel
 open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Tests
 open NUnit.Framework
 
+[<FSharpLanguageLevel(FSharpLanguageLevel.FSharp80)>]
```

Comment:
Do we really want all test cases to use a specific outdated language version going forward? Maybe it'd be better to pin it in some specific cases only?